core/src/main/resources/jenkins/install/pluginSetupWizard.properties

```diff
@@ -56,3 +58,20 @@ installWizard_pluginInstallFailure_message=Some plugins failed to install proper
 with the failed plugins
 installWizard_continue=Continue
 installWizard_retry=Retry
+installWizard_upgradePanel_title=Upgrade
+installWizard_upgradePanel_banner=Welcome to Jenkins {0}!
+installWizard_upgradePanel_message=Jenkins {0} includes some great new features that we think you'll love, install these additional plugins to take advantage of them!
+installWizard_upgradePanel_recommendedActionTitle=Install Suggested Plugins
+installWizard_upgradePanel_recommendedActionDetails=Install the suggested plugins
+installWizard_upgradePanel_skipRecommendedPlugins=No thanks
+installWizard_upgradeComplete_title=Upgrade
+installWizard_pluginsInstalled_banner=Welcome to Jenkins {0}!
+installWizard_upgradeComplete_message=Congratulations! You have upgraded to Jenkins {0}.</p><p>To learn more about the great new features, visit <a target="_blank" href="https://jenkins.io/2.0/">the Jenkins website</a>!
+installWizard_upgradeSkipped_title=Upgrade
+installWizard_upgradeSkipped_banner=Features Not Installed
+installWizard_upgradeSkipped_message=<div class="alert alert-warning fade in">No new plugins will be installed</div> \
```

Comment:
> No new plugins will be installed

I think that this could be confusing, I wouldn't want somebody to assume that the ability to install new plugins has been disabled.

I suggest:

> Suggested plugins will not be installed.